src/utils/react-query-factory.js

```diff
@@ -0,0 +1,93 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Shouldn't need to disable this ESlint error. If the dependency is installed in package.json and you run `npm i`, this ESLint warning shouldn't throw?